bin/database.py

```diff
@@ -96,7 +96,21 @@ def process_cursor(cursor, closure):
     logging.info('Proccessing {} items in cursor ...'.format(cursor.count()))
 
     failed = False
+    cursor_size = cursor.count()
+    cursor_index = 0.0
+    next_percent = 5.0
+    percent_increment = 5
+    if(cursor_size < 20):
+        next_percent = 25.0
+        percent_increment = 25
+    if(cursor_size < 4):
+        next_percent = 50.0
+        percent_increment = 50
     for document in cursor:
```

Comment:
Instead of keeping track of a count, you can use python's `enumerate` to get the cursor's index.

A little more info/description about why `enumerate` is more 'pythonic':
http://stackoverflow.com/questions/522563/accessing-the-index-in-python-for-loops

I try not to be too interested in what looks 'pythonic' and what doesn't, but `enumerate` is a handy tool to use in situations like this.